queue_services/entity-emailer/src/entity_emailer/email_templates/INVOL-DIS-STAGE-1.html

```diff
@@ -71,8 +80,15 @@
             [[whitespace-16px.html]]
 
             <p>
-              You can ask for more time by requesting a delay of the dissolution or 
-              cancellation process. Go to your <a href="{{ entity_dashboard_url }}">business Page</a> and click on “Request 
+              You can ask for more time by requesting 
+              {% if furnishing_name == 'DISSOLUTION_COMMENCEMENT_NO_AR_XPRO' %}
+              a delay of the cancellation process. 
+              {% elif extra_provincials %}
+              a delay of the dissolution or cancellation process. 
+              {% else %}
+              a delay of the dissolution process. 
+              {% endif %}
```

Comment:
Previous work was done improperly at this part. The wordings in design are different for BC companies with EP registration, BC companies without EP registration, and extraprovincial companies.